CHANGELOG.md

```diff
@@ -18,6 +19,7 @@
 🥷 *Internal*
 
 📃 *Docs*
+* "Remote Workflows" updated to describe logging in with a specific runtime, and reflect the current login process (automatic opening of login page, copying of token).
```

Comment:
This change is not a mistake - its a fix for other PR that mistakely put this entry into already rolled-out changelog